flume/README.md

```diff
@@ -398,6 +398,7 @@ cygnusagent.sinks.hdfs-sink.channel = hdfs-channel
 # sink class, must not be changed
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-connectors/commit/e820c273f2a3b1bbac288b3c750d48144d7572c8